src/network/clientpackethandler.cpp

```diff
@@ -994,14 +994,22 @@ void Client::handleCommand_AddParticleSpawner(NetworkPacket* pkt)
 	p.amount             = readU16(is);
 	p.time               = readF32(is);
 
-	// older protocols do not support tweening, and send only
-	// static ranges, so we can't just use the normal serialization
-	// functions for the older values.
-	p.pos.start.legacyDeSerialize(is);
-	p.vel.start.legacyDeSerialize(is);
-	p.acc.start.legacyDeSerialize(is);
-	p.exptime.start.legacyDeSerialize(is);
-	p.size.start.legacyDeSerialize(is);
+	bool missing_end_values = false;
+	if (m_proto_ver >= 42) {
```

Comment:
New serialization of particle spawner parameters should probably be documented in networkprotocol.h

Reply:
Done.